### PR TITLE
Cleanup user projection and fix/rm tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: tests/system/action/projector/test_add_to_preview.py [hunk 1]
```diff
@@ -106,26 +106,6 @@ def test_add_to_preview_check_meeting_id(self) -> None:
             response.json["message"],
         )
 
-    def test_add_to_preview_user(self) -> None:
```
Comment (jsangmeister) [inline tests/system/action/projector/test_add_to_preview.py:109]: Keep the test and assert that it fails

File: tests/system/action/projector/test_project.py [hunk 2]
```diff
@@ -359,41 +343,6 @@ def test_meeting_as_content_object_ok(self) -> None:
         )
         self.assert_status_code(response, 200)
 
-    def test_user_as_content_object_okay(self) -> None:
```
Comment (jsangmeister) [inline tests/system/action/projector/test_project.py:362]: same here

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: tests/system/action/projector/test_add_to_preview.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/projector/test_add_to_preview.py:109]: Okay. Done.

File: tests/system/action/projector/test_project.py (hunk 2 above)
Comment (reiterl) [reply tests/system/action/projector/test_project.py:362]: Okay. Done.

File: tests/system/action/projector/test_project.py [hunk 3]
```diff
@@ -373,25 +357,10 @@ def test_user_as_content_object_okay(self) -> None:
...
-        self.assert_model_exists(
-            "projection/112",
-            {
-                "content_object_id": "user/2",
-                "current_projector_id": 75,
-                "stable": False,
-            },
+        self.assert_status_code(response, 400)
+        assert (
+            "he collection 'user' is not available for field 'content_object_id' in collection 'projection'."
```
Comment (jsangmeister) [inline tests/system/action/projector/test_project.py:362]:
```suggestion
            "The collection 'user' is not available for field 'content_object_id' in collection 'projection'."
```

**Review by jsangmeister — CHANGES_REQUESTED** [review]
minor typo, otherwise fine. Can be merged afterwards

File: tests/system/action/projector/test_project.py (hunk 3 above)
Comment (reiterl) [reply tests/system/action/projector/test_project.py]: Done.

**Comment by reiterl** [comment]
@jsangmeister merging is blocked, why that?

**Review by jsangmeister — APPROVED** [review]

**Comment by jsangmeister** [comment]
@reiterl because there were still changes requested, it seems

**Comment by reiterl** [comment]
Is that new? How has added that feature?

**Comment by jsangmeister** [comment]
I recently changed the settings to require a PR and prevent directly pushing to main. It seems like GH automatically requires all change requests to be resolved then and I don't see any way to change this, but I don't think its much of a problem.

**Pull request merged**